contracts/CollectiveGovernance.sol

```diff
@@ -65,6 +65,17 @@ import "../contracts/TimeLock.sol";
 ///
 /// @dev The VoterClass is common to all proposed votes as are the project supervisors.   Individual supervisors may
 /// be configured as part of the proposal creation workflow but project supervisors are always included.
+
+error CG__VotingNotReady();
```

Comment:
Please move these errors into the Governance.sol:Governance interface.   That will allow you to remove the 'Prefix'.

Also if you feel comfortable enough add a comment about the meaning of each error.   Otherwise I will add it after this change gets merged.